tests/integration/targets/rds_cluster_param_group/tasks/main.yaml

```diff
@@ -0,0 +1,255 @@
+      amazon.aws.rds_cluster_param_group_info:
+        name: "{{ rds_cluster_param_group_name }}"
+      register: cluster_params
+
+    - name: Validate that the resource has been updated with tags
+      assert:
+        that:
+          - create_tag is changed
+          - cluster_params.db_cluster_parameter_groups.0.tags == resource_tags
+
+    # Test adding tag to existing RDS cluster parameter group (idempotency)
+    - name: Update existing RDS cluster parameter group with tags (idempotency)
+      amazon.aws.rds_cluster_param_group:
+        name: "{{ rds_cluster_param_group_name }}"
+        db_parameter_group_family: "{{ dbparam_group_family }}"
+        description: "RDS cluster param group for Engine {{ engine_versions.db_engine_versions.0.engine_version }}"
+        tags: "{{ resource_tags }}"
+      register: create_tag_idempotency
+
+    - name: Describe RDS parameter group
+      amazon.aws.rds_cluster_param_group_info:
+        name: "{{ rds_cluster_param_group_name }}"
+      register: cluster_params
+
+    - name: Validate that the resource has been updated with tags
+      assert:
+        that:
+          - create_tag_idempotency is not changed
+          - cluster_params.db_cluster_parameter_groups.0.tags == resource_tags
+
+    # Test adding invalid or not modifiable parameters
+    - name: Update existing RDS cluster parameter group with tags (idempotency)
+      amazon.aws.rds_cluster_param_group:
+        name: "{{ rds_cluster_param_group_name }}"
+        db_parameter_group_family: "{{ dbparam_group_family }}"
+        description: "RDS cluster param group for Engine {{ engine_versions.db_engine_versions.0.engine_version }}"
+        parameters:
+          archive_library: "testing_lib"  # This parameter is not modifiable
+          invalid_fake: true  # this parameter does not exist
+      register: update_parameters
+      ignore_errors: true
+
+    - name: Ensure module failed to update invalid/not modifiable parameters
+      assert:
+        that:
+          - update_parameters is failed
+          - update_parameters.msg == error_msg
+      vars:
+        error_msg: "The following parameters are not modifiable: archive_library. The following parameters are not available parameters for the 'postgres16' DB parameter group family: invalid_fake."
+
+    # Test adding parameters
+    - name: Modify RDS cluster parameter group with new parameters
+      amazon.aws.rds_cluster_param_group:
+        name: "{{ rds_cluster_param_group_name }}"
+        db_parameter_group_family: "{{ dbparam_group_family }}"
+        description: "RDS cluster param group for Engine {{ engine_versions.db_engine_versions.0.engine_version }}"
+        parameters:
+          array_nulls: 0
+          authentication_timeout: 50
+      register: update_parameters
+
+    - name: Ensure module failed to update invalid/not modifiable parameters
+      assert:
+        that:
+          - update_parameters is changed
```

Comment:
Could we include a check to verify the updated parameter value?

Reply:
updated